src/aerialmap_display.cpp

```diff
@@ -494,7 +483,7 @@ void AerialMapDisplay::transformAerialMap()
   static std::string const mapFrame = "map";
```

Comment:
could you add a comment that mentions why we have this constant? -> [REP 105 ](https://www.ros.org/reps/rep-0105.html)

Reply:
Let's do this in the PR for #54 and not in this PR